draft-ietf-ppm-dap.md

```diff
@@ -1370,7 +1370,7 @@ struct {
 
 struct {
   ReportShare report_share;
-  opaque payload<0..2^32-1>;
+  opaque payload<10..2^32-1>;
```

Comment:
I think the 10-byte minimum is incorrect; see https://github.com/cfrg/draft-irtf-cfrg-vdaf/pull/428/files#r1765155695.

Reply:
Done, good catch.